### PR TITLE
Add * as default value for comma-separated lists that support it

### PR DESCRIPTION
**Pull request by artem-shelkovnikov**
## Part of https://github.com/elastic/enterprise-search-team/issues/7205

This PR adds "*" as a default value for connector fields that support comma-separated values and asterisk. Affected connectors/fields are:

- MSSQL connector `tables`
- MySQL connector `tables`
- Oracle connector `tables`
- PostgreSQL connector `tables`
- Redis connector `database`
- ServiceNow connector `services`
- Sharepoint Online connector `site_collections`

#### Pre-Review Checklist
- [x] this PR does NOT contain credentials of any kind, such as API keys or username/passwords (double check `config.yml.example`)
- [x] this PR has a meaningful title
- [x] this PR links to all relevant github issues that it fixes or partially addresses
- [x] if there is no GH issue, please create it. Each PR should have a link to an issue
- [x] this PR has a thorough description
- [x] Tested the changes locally
- [x] Added a label for each target release version (example: `v7.13.2`, `v7.14.0`, `v8.0.0`)
- [x] if you added or changed Rich Configurable Fields for a Native Connector, you made a corresponding PR in [Kibana](https://github.com/elastic/kibana/blob/main/packages/kbn-search-connectors/types/native_connectors.ts)

## Related Pull Requests

https://github.com/elastic/kibana/pull/182957

### CODE REVIEW

**Review by navarone-feekery — DISMISSED** [review]
I love this, nice one!

File: share/man/man1/ttx.1 [hunk 1]
```diff
@@ -0,0 +1,225 @@
+.Dd May 18, 2004
```
[the remark this answers is not in the document]
Comment (artem-shelkovnikov) [reply share/man/man1/ttx.1:1]: Idk how it got here, wat

[the remark this answers is not in the document]
Comment (artem-shelkovnikov) [reply share/man/man1/ttx.1:1]: Just removed

[the remark this answers is not in the document]
Comment (navarone-feekery) [reply share/man/man1/ttx.1:1]: It's still here 👻

[the remark this answers is not in the document]
Comment (navarone-feekery) [reply share/man/man1/ttx.1]: AH now it's gone, ignore ^

**Review by navarone-feekery — APPROVED** [review]

**Pull request merged**